docs/Doxyfile

```diff
@@ -972,6 +972,7 @@ WARN_LOGFILE           =
 # Note: If this tag is empty the current directory is searched.
 
 INPUT                  =
+# {{INPUT}}
```

Comment:
I'm not super familiar with doxygen templates. Why are these additions necessary?

Reply:
https://github.com/TendTo/rules_doxygen/blob/main/doxygen/doxygen.bzl#L77 is the magic.  That does the replacement.